doc/releases/changelog-0.27.0.md

```diff
@@ -174,32 +256,112 @@
 
   Note that this change depends on `jax.pure_callback`, which requires `jax>=0.3.17`.
 
-* The `QNode` class now accepts an `auto` interface, which automatically detects the interface of the given input.
-  [(#3132)](https://github.com/PennyLaneAI/pennylane/pull/3132)
+<h4>The `qml.pauli` module 👀</h4>
 
-  ```python
-  dev = qml.device("default.qubit", wires=2)
-  @qml.qnode(dev, interface="auto")
-  def circuit(weight):
-      qml.RX(weight[0], wires=0)
-      qml.RY(weight[1], wires=1)
-      return qml.expval(qml.PauliZ(0))
+* We've reorganized and grouped everything in PennyLane responsible for manipulating Pauli operators into a `pauli` module. The `grouping` module has been deprecated as a result, and logic was moved from `pennylane/grouping` to `pennylane/pauli/grouping`.
+  [(#3179)](https://github.com/PennyLaneAI/pennylane/pull/3179)
 
-  interface_tensors = [[0, 1], np.array([0, 1]), torch.Tensor([0, 1]), tf.Variable([0, 1], dtype=float), jnp.array([0, 1])]
-  for tensor in interface_tensors:
-      res = circuit(weight=tensor)
-      print(f"Result value: {res:.2f}; Result type: {type(res)}")
+* `qml.pauli.PauliWord` and `qml.pauli.PauliSentence` can be used to represent tensor products and linear combinations of Pauli operators, respectively. These provide a more performant method to compute sums and products of Pauli operators.
```

Comment:
Maybe worth swapping the order of these two points? As a user I wasn't that excited by the reorganization point.

Reply:
I think I want to keep the order as it is. If a user sees `qml.pauli` first, they might be a little confused